blackbird-environment-templates/templates/aes-gw.yaml

```diff
@@ -3,6 +3,9 @@ kind: GatewayClass
 metadata:
   labels:
     app.kubernetes.io/instance: eg-edge-stack
+{{- range $key, $value := .Values.Labels }}
```

Comment:
Modern engineering prioritizes readability/maintainability first.
Altering external manifests, particularly with this level of repetition isn't an acceptable pattern.

If this goes into place, I'm just going to overwrite these with never versions later and these will be removed.
And in this case that's irregardless of whether we have tooling to manually apply these alterations.

Reply:
Matt's point (which is a good one) is to ensure we have an easy way for a customer to know what we've installing. That doesn't imply a solution and it doesn't imply this solution.

As an organization (or any organization for that matter), we need to remove maintenance concerns, not add to them.

This is also an additional story item that can (and should) be done iteratively and separately. Even if it's considered a requirement.